examples/bigquery-row-access-groups/schedule_job.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PROJECT_ID=$1
+REGION=us-central1
+
+gcloud beta scheduler jobs create http group_sync \
+  --project $PROJECT_ID \
+  --schedule="11 * * * *" \
```

Comment:
Any reason for running every 11 minutes? I think this might be a typo to run every minute to quickly update row level security when iam changes?

Reply:
This is not running every 11 minutes, it's once every hour on the 11th minute (just an arbitrary value I chose). Since this is something a user is likely to change, I pulled it out into a variable.